scalyr_agent/copying_manager.py

```diff
@@ -821,7 +833,12 @@ def run(self):
 
                             # Now block for the response.
                             blocking_response_time_start = time.time()
-                            (result, bytes_sent, full_response) = get_response()
+                            if self.__config.use_new_ingestion:
+                                result = "success"
+                                bytes_sent = 0
```

Comment:
I had a look again and the ``SendEventsAPIResponse`` message already contains ``bytes_ingested`` field which you could use.

This indicates actual bytes ingested though so if we are interested in raw number of bytes sent (pre compression), I will modify a client to expose this information on ``send_events``.

Reply:
This may help - https://github.com/scalyr/python-ingestion-library/pull/12

Keep in mind that ``headers_pb.events_bytes`` field only includes total bytes which represent the size of ``attributes`` field for all the events in the payload - aka it doesn't include gRPC metadata + headers overhead (which is probably fine for here).

Reply:
Yeah the raw number of bytes is what we care about, its needed for the rate limiting mostly.